src/com/projectkorra/projectkorra/configuration/ConfigManager.java

```diff
@@ -1243,6 +1243,7 @@ public static void configCheck(final ConfigType type) {
 			config.addDefault("Abilities.Earth.EarthGrab.DamageThreshold", 4);
 
 			config.addDefault("Abilities.Earth.EarthTunnel.Enabled", true);
+			config.addDefault("Properties.Earth.EarthTunnel.RevertCheckTime", 30000);

```

Comment:
this should still follow the convention of `Abilities.<element>.<ability>.<option>` so the path should be `Abilities.Earth.EarthTunnel.RevertCheckTime`, reason being so we keep all the ability-specific information within the ability's section of the config.